capella_console_client/repeat_request.py

```diff
@@ -87,6 +88,15 @@ def create_repeat_request(
     if product_types is not None:
         payload["properties"]["processingConfig"] = {"productTypes": product_types}
 
+    if contract_id:
```

Comment:
Since the API will be handling this validation ("contract not found") I recommend to drop this validation piece both here and tasking_request.py::create_tasking_request simplifying this down to

```python
if contract_id:
    payload["contractId"] = contract_id
```

note the camelCase above

Reply:
> note the camelCase above.

Is the REST API accepting this as `contractId` now? IIRC I was using `contract_id` while testing few weeks back. Is it the change in the schema there or both of them are actually valid?